### PR TITLE
Add beginner friendly accessibility questions

### PR DESCRIPTION
**Pull request by THEFZNKHAN**
## Summary of changes

Add six more questions in the beginner-friendly accessibility questions.
Mentioned in issue #899

## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -1,4 +1,64 @@
...
+    Answer: "Clarity",
+    Distractor1: "Interactivity",
+    Distractor2: "Animation",
+    Distractor3: "Multimedia",
+    Explanation:
+      "Semantic HTML tags provide clear and meaningful structure to web content.",
+    Link: "https://webaim.org/techniques/semanticstructure"
+  },
+  {
+    Question: "What is the role of labels in accessibility?",
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:43]:
```suggestion
    Question: "What is the role of a label in accessibility?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by THEFZNKHAN** [comment]
I change the "labels" to "label"
If any other problem than tell me 😊

**Comment by THEFZNKHAN** [comment]
Sorry for too many commits 
But I do typos by mistake.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**